exampleCourse/questions/demo/computeRotationMatrix/server.py

```diff
@@ -14,11 +14,11 @@ def generate(data):
     M[1, 0] = math.sin(math.pi / a)
     M[1, 1] = math.cos(math.pi / a)
     if a < 0:
-        data["params"]["a"] = "- \\frac{\pi}{" + str(-a) + "}"
-        data["params"]["b"] = "R_{-\\frac{\pi}{" + str(-a) + "}}"
+        data["params"]["a"] = "- \\frac{\\pi}{" + str(-a) + "}"
```

Comment:
Can we make these raw strings instead of using escape characters? Maybe also worth using a format string here.

Reply:
With a format string we'd need to use `{{{` for the literal curly braces, which IMO is less readable than just doing string concatenation. And I'm also inclined to not use r-strings for the potentially stupid reason that my editor formats them in such a way as to still make `\f` seem like an escaped character:

<img width="424" alt="Screenshot 2023-01-27 at 16 58 34" src="https://user-images.githubusercontent.com/1476544/215232552-179e4f7a-1516-498d-ba8c-8b0ad2b12867.png">

Reply:
That's very weird, what editor are you using? VS code seems to be ok with raw strings.

Reply:
This is VSCode too 👀 maybe it's a rogue extension?

Reply:
Do you have pylance installed?

Reply:
I do!

Reply:
May be related to this: https://github.com/microsoft/vscode-python/issues/13702

Reply:
Yep, alas, it's this issue: https://github.com/MagicStack/MagicPython/issues/114

Reply:
Ahhh, yeah I guess go with capital R